lib/webpack-cli.js

```diff
@@ -152,19 +157,12 @@ class WebpackCLI extends GroupHelper {
 
     async runCommand(command, ...args) {
         // TODO: rename and depreciate init
-        if (command.name === 'create') {
-            command.name = 'init';
-        } else if (command.name === 'loader') {
-            command.name = 'generate-loader';
-        } else if (command.name === 'plugin') {
-            command.name = 'generate-plugin';
-        }
-        return await require('./commands/external').run(command.name, ...args);
+        return await require('./commands/external').run(this.commandNames[command.name], ...args);
     }
 
     runHelp(args) {
         const HelpGroup = require('./groups/help');
-        const names = require('./utils/commands').names;
```

Comment:
What's the reason of this change?

Reply:
Object destructuring assignment makes the respective code snippet much concise.